less/admin/ExtensionWidget.less

```diff
@@ -0,0 +1,58 @@
+.ExtensionsWidget-title {
+  color: @muted-color;
+}
+
+.ExtensionsWidget-list {
+  > .container {
+    padding: 0;
+  }
+}
+.ExtensionGroup {
+  margin-bottom: 20px;
+
+  h3 {
+    color: @muted-color;
+    text-transform: uppercase;
+    font-size: 12px;
+    margin: 0 0 10px;
+  }
+}
+
+.ExtensionList {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  .clearfix();
+
+  > li {
+    float: left;
+    text-align: left;
+    position: relative;
+  }
+}
```

Comment:
The extension widget gets very messy on different screen sizes. I would really prefer we avoid floats and instead go for flexbox (https://caniuse.com/mdn-css_types_flex)

It makes the list a lot more resilient and dynamic, while keeping it nice looking.

locally this is what I had to edit (it's in css but.. yeah)
```css
.ExtensionsWidget-title {
    color: #667c99;
    margin-top: 0;
}
.ExtensionList {
    margin: 0 -6px;
    padding: 0;
    list-style: none;
    display: flex;
    flex-wrap: wrap;
}
.ExtensionList>li {
    text-align: left;
    position: relative;
    display: block;
}
.ExtensionListItem {
    width: 90px;
    height: 90px;
    margin: 6px;
}
```

PS: also had to drop the ::before and ::after pseudo elements from `.ExtensionList`

Reply:
Even better, could we go for CSS grid?

Reply:
oh yes! CSS grid would most likely be much better, something like ?

```css
.ExtensionList {
    display: grid;
    grid-gap: 10px;
    grid-template-columns: repeat(auto-fit, 90px);
}
```